en/04_Changelogs/4.12.0.md

```diff
@@ -267,14 +270,20 @@ Silverstripe CMS 4 includes multiple tasks to migrate files to newer states. The
 
 - `extra_fields` data for a [ManyManyList](api:SilverStripe\ORM\ManyManyList) can now be set easily with the ORM - simply call the [setExtraData()](api:SilverStripe\ORM\ManyManyList::setExtraData()) method. See [the `many_many` documentation](/developer_guides/model/relations#many-many) for more details.
 
-## Bugfixes {#bugfixes}
+## Deprecating APIs ahead of Silverstripe CMS 5 {#deprecation-cms5}
```

Comment:
This seems appropriate as its own section. I thought about explicitly encouraging people to enable deprecation warning, but given the poor output of those warnings right now, it seems better to just leave that for 4.13.